native/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxTreeGenerator.java

```diff
@@ -259,8 +396,10 @@ public static SyntaxTree generateClientSyntaxTree(Entity entity) {
         return balTree.modifyWith(modulePartNode);
     }
 
+
     private static Class createClientClass(Entity entity, String className, List<Node> subFields,
-                                           HashMap<String, String> keys, String keyType, boolean keyAutoInc) {
+                                           List<Node> joinSubFields, HashMap<String, String> keys,
```

Comment:
Shall we breakdown the `createClientClass()` functions into separate functions for `update()`, `read()`, `delete()` functions? Seems this function is too lengthy to read.

Reply:
updated in [16404a1](https://github.com/ballerina-platform/persist-tools/pull/23/commits/16404a1add767f483c804d56de2b5856368650e1)